global_helpers/panther_greynoise_helpers.py

```diff
@@ -54,6 +54,13 @@ def actor(self, match_field) -> str:
     def url(self, match_field) -> str:
         return f"https://www.greynoise.io/viz/ip/{deep_get(self.noise, match_field, 'ip')}"
 
+    def context(self, match_field) -> dict:
+        return{
+            "IP": self.ip_address(match_field),
+            "Classification": self.classification(match_field),
+            "actor": self.actor(match_field),
+            "GreyNoise_URL": self.url(match_field),
```

Comment:
curious why are we mis-casing the keys? (some start with lower some start with cap)

Reply:
Oh, not intentional, thank you

Reply:
Fixed